api-server/src/main/java/vermillion/http/HttpServerVerticle.java

```diff
@@ -44,1048 +44,1139 @@
 
 public class HttpServerVerticle extends AbstractVerticle {
 
-  public final Logger logger = LoggerFactory.getLogger(HttpServerVerticle.class);
```

Comment:
Use google-java-format and update this file. The spacing is completely messed up.

Reply:
Use palantir-java-format. I've made the necessary changes